WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -338,34 +340,55 @@ - (void)validateBlogDetailsViewController
     }
 }
 
+- (void)autoSyncBlogs
+{
+    if (!self.lastSyncDate || fabs([self.lastSyncDate timeIntervalSinceNow]) > AutoSyncInterval) {
+        [self syncBlogs];
```

Comment:
I think we should be calling `[self.tableView.refreshControl beginRefreshing]` from here?

Reply:
According to [the docs](https://developer.apple.com/documentation/uikit/uirefreshcontrol/1624842-beginrefreshing):

> Call this method when an external event source triggers a programmatic refresh of your scrolling view. In a table view, for example, if you use an instance of Timer to refresh the contents of the table view periodically, you would call this method as part of your timer handler. This method updates the state of the refresh control to reflect the in-progress refresh operation. When the refresh operation ends, be sure to call the endRefreshing() method to return the control to its default state.

I understand what this does is telling the refresh control that we are already syncing. That way if the user starts dragging to refresh, a loading indicator would appear instead of triggering a second refresh (or trying to).